packages/react-ui-map/src/Mapbox/index.js

```diff
@@ -57,8 +57,23 @@ export default class Mapbox extends PureComponent {
 
       sources.forEach((v, i) => {
         map.addSource(sources[i].id, sources[i].source)
+      })
+
+      layers.forEach((v, i) => {
         map.addLayer(layers[i].layer)
       })
+
+      map.on('mouseenter', 'markers-default', () => {
+        map.getCanvas().style.cursor = 'pointer';
```

Comment:
i dont think we need this mouse events - what happens without them? does the marker show a cursor?

cursors are not required for mobile map anyways